test/testinput/bufr_read_std_wmo_radiosonde.yaml

```diff
@@ -0,0 +1,82 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./testinput/20210203.radiosonde"
+      isStdFormat: true
+      tablepath: "./testinput/bufr_tables"
+
+      mnemonicSets:
+        - mnemonics: [CLATH, CLONH]
+        - mnemonics: [TMDB, TMDP, WDIR, WSPD]
+          channels: 1-5880
```

Comment:
This illustrates why 'channels' is a bad word to use here... but tackle that in a future PR, obviously.